test/project/git/GitProjectTest.ts

```diff
@@ -177,24 +182,10 @@ describe("GitProject", () => {
             .then(p => {
                 checkProject(p);
                 const gp: GitProject = GitCommandGitProject.fromProject(p, Creds);
-                const baseDir = (p as LocalProject).baseDir;
-
-                gp.checkout(sha)
-                    .then(_ => {
-                        exec.exec("git status", {cwd: baseDir}, (err, stdout, stderr) => {
-                            if (err) {
-                                // node couldn't execute the command
-                                console.error(`Node err on dir [${baseDir}]: ${err}`);
-                                return;
-                            }
-
-                            // the *entire* stdout and stderr (buffered)
-                            console.log(`stdout: ${stdout}`);
-                            console.log(`stderr: ${stderr}`);
-                            done();
-                        });
-                    });
-            }).catch(done);
+                // TODO: check that it has the right sha
+                // this will be easy after cached-clones-58 is merged
```

Comment:
So removing this test is intentional but temporary?

Reply:
the test wasn't actually testing anything, just running a command and printing the output.
soon we'll be able to have it test something